### DISCUSSION TITLE
Allow null values in config

### DISCUSSION THREAD

**Discussion by mijewe** [trendyminds/design-tokens#4] [Ideas]
I would like to be able to set a null value in my .json config, so I can do a simple conditional in my template.

For example, I have a field `spaceAbove` that uses the following config:
```
{
	"None": null,
	"Small": "sm",
	"Medium": "md",
	"Large": "lg"
}
```

and then in my template I want to:
```
{% set spaceAboveClass = spaceAbove ? "pt-#{spaceAbove}" : null %}
```

whereas currently I can't because `spaceAbove` isn't `null` - it's a DesignToken object.

I can work around this by setting the value to "none" and doing a check on `spaceAbove != "none"`, but it's not very tidy.

**Comment by furioursus** [comment]
Hey @mijewe, i actually get around this by just using an empty string to achieve the same thing.

Additionally, you should actually be utilizing these keys to store the actual class, not a shorthand value, otherwise the value of the plugin is lost and you'd do just as well with a regular dropdown field.

**Comment by mijewe** [comment]
Oh yeah that's a good idea - I'll try that thanks

Well in this case I'm using the same values for two dropdowns in the CMS - spaceAbove and spaceBelow, and use those dropdowns in multiple places in various matrix blocks, so I still get the DRY aspect of the plugin.

Though I guess I could use nested keys for above and below - good call
